cmd/ocitools/validate.go

```diff
@@ -381,6 +386,47 @@ func checkLinux(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string)
 	return
 }
 
+func checkLinuxResources(r rspec.Resources, hostCheck bool) (msgs []string) {
+	logrus.Debugf("check linux resources")
+
+	if r.OOMScoreAdj != nil && (int(*r.OOMScoreAdj) < -1000 || int(*r.OOMScoreAdj) > 1000) {
+		msgs = append(msgs, fmt.Sprintf("OOMScoreAdj range is from -1000 to 1000"))
```

Comment:
On Wed, Aug 17, 2016 at 10:27:06PM -0700, Ma Shimiao wrote:
> Maybe they are some short of thing, but Ocitools and runtime schema
> are two projects and ocitools does not have these yet.

ocitools depends strongly on runtime-spec already, both for the
Markdown spec and the Go types.  I don't see any new issues if we
start depending on the JSON Schema it provides as well.  I'll file a
PR that adds a JSON Schema initial validation pass tonight.